tests/coalaTest.py

```diff
@@ -323,3 +323,16 @@ def test_coala_without_color(self):
                 self.assertRegex(err, '^\[WARNING\]')
             self.assertEqual(
                 retval, 0, 'coala must return zero when there are no errors')
+
+    def test_coala_ignore_file(self):
+        with bear_test_module(), \
+                prepare_file(['#fixme  '], None) as (lines, filename):
+            retval, stdout, stderr = execute_coala(
+                    coala.main, 'coala',
+                    '--non-interactive',
+                    '-f', re.escape(filename),
+                    '-i', re.escape(filename),
+                    '-b', 'ErrorTestBear')
```

Comment:
Don't use `ErrorTestBear`. This bear is designed to test coala handling of errors due to missing runtime deps. Instead please use `LineCountTestBear` and check stdout to make sure no output is found for that file.